fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/api/object/builder/v1/block/entity/FabricBlockEntityTypeBuilder.java

```diff
@@ -31,23 +35,36 @@
  */
 public final class FabricBlockEntityTypeBuilder<T extends BlockEntity> {
 	private final Factory<? extends T> factory;
-	private final Block[] blocks;
+	private final List<Block> blocks;
 
-	private FabricBlockEntityTypeBuilder(Factory<? extends T> factory, Block[] blocks) {
+	private FabricBlockEntityTypeBuilder(Factory<? extends T> factory, List<Block> blocks) {
 		this.factory = factory;
 		this.blocks = blocks;
 	}
 
 	public static <T extends BlockEntity> FabricBlockEntityTypeBuilder<T> create(Factory<? extends T> factory, Block... blocks) {
-		return new FabricBlockEntityTypeBuilder<>(factory, blocks);
+		List<Block> blocksList = new ArrayList<>(blocks.length);
+		Collections.addAll(blocksList, blocks);
+
+		return new FabricBlockEntityTypeBuilder<>(factory, blocksList);
+	}
+
+	public FabricBlockEntityTypeBuilder<T> addBlock(Block block) {
+		this.blocks.add(block);
+		return this;
+	}
+
+	public FabricBlockEntityTypeBuilder<T> addBlocks(Block... blocks) {
```

Comment:
```suggestion
	/**
	 * Adds supported blocks for the block entity type.
	 *
	 * @param blocks the supported blocks
	 * @return this builder
	 */
	public FabricBlockEntityTypeBuilder<T> addBlocks(Block... blocks) {
```

Reply:
Fixed in ca96055b02d12a3c1a428fc12098d3e6f79d98fb